starlite/plugins/tortoise_orm.py

```diff
@@ -22,18 +23,71 @@
     raise MissingDependencyException("tortoise-orm is not installed") from e
 
 if TYPE_CHECKING:
+    from typing import Any, Mapping
+
     from pydantic_openapi_schema.v3_1_0 import Schema
     from typing_extensions import TypeGuard
 
+    from starlite.enums import RequestEncodingType
+
+
+TortoiseModelT = TypeVar("TortoiseModelT", bound="Model")
+
 
-class TortoiseORMPlugin(SerializationPluginProtocol[Model, BaseModel], OpenAPISchemaPluginProtocol[Model]):
+class TortoiseORMPlugin(SerializationPluginProtocol[Model, PydanticModel], OpenAPISchemaPluginProtocol[Model]):
```

Comment:
can prob just extract out a mixin class to dedup the code between here and `FromPydantic`